pkg/release/config/client.go

```diff
@@ -22,12 +22,19 @@ func endpoint(c api.Candidate) string {
 type Job struct {
 	Name                 string `json:"name"`
 	api.MetadataWithTest `json:",inline"`
+
+	AggregatedCount int `json:"aggregatedCount,omitempty"`
+}
+
+type AggregatedJob struct {
+	AnalysisJobCount int `json:"analysisJobCount"`
 }
 
 type Verify map[string]VerifyItem
 
 type VerifyItem struct {
-	ProwJob Job `json:"prowJob"`
+	ProwJob           Job           `json:"prowJob"`
+	AggregatedProwJob AggregatedJob `json:"aggregatedProwJob,omitempty"`
```

Comment:
This can actually be empty. This struct is being used to marshal the release controller's response to an object. The response can contain jobs that don't have the `AggregatedProwJob` field.

Reply:
I updated the code, and added also a testcase where this field is empty.

Reply:
This is what i meant.
https://www.sohamkamani.com/golang/omitempty/#values-that-cannot-be-omitted

Reply:
updated. Thanks!